docs/tutorial.md

```diff
@@ -343,46 +352,60 @@ Some of the new styles have more than one selector separated by a space. The spa
 }
 ```
 
-The `.started` selector matches any widget with a `"started"` CSS class. While `#start` matches a child widget with an ID of `"start"`. So it matches the Start button only for Stopwatches in a started state.
+The `.started` selector matches any widget with a `"started"` CSS class. While `#start` matches a child widget with an ID of `"start"`.
+When combined with a space, the the selector will match the start button *only* if it is inside a container with a CSS class of "started".
 
-The rule is `"display: none"` which tells Textual to hide the button.
+As before, the `display: none` rule will cause any matching widgets to be hidden from view. 
+
+If we were to write this in English, it would be something like: "Hide the start button if the widget is already started".
 
 ### Manipulating classes
 
 Modifying a widget's CSS classes is a convenient way to update visuals without introducing a lot of messy display related code.
 
-You can add and remove CSS classes with the [add_class()][textual.dom.DOMNode.add_class] and [remove_class()][textual.dom.DOMNode.remove_class] methods. We will use these methods to connect the started state to the Start / Stop buttons.
+You can add and remove CSS classes with the [add_class()][textual.dom.DOMNode.add_class] and [remove_class()][textual.dom.DOMNode.remove_class] methods.
+We will use these methods to connect the started state to the Start / Stop buttons.
 
-The following code will start or stop the stopwatches in response to clicking a button.
+The following code will start or stop the stopwatches in response to clicking a button:
 
 ```python title="stopwatch04.py" hl_lines="13-18"
 --8<-- "docs/examples/tutorial/stopwatch04.py"
 ```
 
-The `on_button_pressed` method is an *event handler*. Event handlers are methods called by Textual in response to an *event* such as a key press, mouse click, etc. Event handlers begin with `on_` followed by the name of the event they will handle. Hence `on_button_pressed` will handle the button pressed event.
+The `on_button_pressed` method is an *event handler*. Event handlers are methods called by Textual in response to an *event* such as a key press, mouse click, etc.
+Event handlers begin with `on_` followed by the name of the event they will handle.
+Hence `on_button_pressed` will handle the button pressed event.
```

Comment:
This felt a bit abrupt relative to the detail elsewhere in the tutorial. I'd assumed after following the tutorial folks could write an event handler, but I don't think there's enough info here to actually learn about event handlers other than just copy-pasting. Not sure if just a link is needed to help direct people to the relevant page in the guide or something else.